playground-common/update-playground-setups.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Helper script to update all playground setups when we make changes
+# on how playground works (in other words, when we update setup-playground.sh).
+function absPath {
+    python3 -c "import os.path; print(os.path.abspath('$1'))"
+}
+
+WORKING_DIR=$(pwd)
+PLAYGROUND_REL_PATH=$(dirname $0)
+SUPPORT_ROOT_ABS_PATH=$(absPath "$PLAYGROUND_REL_PATH/..")
+PLAYGROUND_ABS_PATH=$(absPath $PLAYGROUND_REL_PATH)
+
+# re-runs the playground setup script on the given folder
+function setupPlayground {
+    echo "setting up playground in $1"
+    cd $1
+    $PLAYGROUND_ABS_PATH/setup-playground.sh
+    echo "finished setting up playground in $1"
+}
+
+# find all playground settings files
+PLAYGROUND_SETTINGS_FILES=$(egrep -lir --include=settings.gradle "setupPlayground" $SUPPORT_ROOT_ABS_PATH)
```

Comment:
just `egrep -lr` since i think you don't want case insensitive?

Reply:
Also this only works if you run from `frameworks/support` right? Can we add validation for `$WORKING_DIR` ?

Reply:
it should work anywhere.